test/system/030-run.bats

```diff
@@ -901,4 +901,24 @@ $IMAGE--c_ok" \
     run_podman rm $ctr_name
 }
 
+@test "podman run --privileged as root with systemd will not mount /dev/tty" {
+    skip_if_rootless "this test only makes sense as root"
+
+    ctr_name="container-$(random_string 5)"
+    run_podman run --rm -d --privileged --systemd=always --name "$ctr_name" "$IMAGE" /home/podman/pause
+
+    TTYs=$(ls /dev/tty*|sed '/^\/dev\/tty$/d')
+
+    if [[ $TTYs = "" ]]; then
+        false
```

Comment:
Courtesy alert: it looks like you're used to regular BATS. Regular BATS is horrible for end users; this is not regular BATS. We provide mechanisms to actually make debugging possible. In this case, please replace `false` with:
```bash
    die "Did not find any /dev/ttyN devices on local host"
```
(or something else appropriate)